mscore/startupWizard.cpp

```diff
@@ -92,8 +138,37 @@ StartupWizard::StartupWizard(QWidget* parent)
       setWindowTitle(tr("Startup Wizard"));
 
       p1 = new StartupWizardPage1(this);
+      p2 = new StartupWizardPage2(this);
+      p3 = new StartupWizardPage3(this);
 
       addPage(p1);
+      addPage(p2);
+      addPage(p3);
+
+      connect(p1->getLanguages(), SIGNAL(currentIndexChanged(int)), SLOT(langChanged()));
+      }
+
+void StartupWizard::langChanged()
+      {
+      p2->setCurrentLayout();
+      }
+
+void StartupWizardPage2::setCurrentLayout()
+      {
+      QString langName = QWizardPage::field("langName").toString();
```

Comment:
Also, take a look to http://doc.qt.io/qt-5/qcombobox.html#setItemData 
Using data, instead of the combo text could be better, especially in this case.